src/components/AboutNFTPage/NFTWidgetSection.vue

```diff
@@ -0,0 +1,277 @@
+              cx="-141.37"
+              cy="580.13"
+              fx="-141.37"
+              fy="580.13"
+              r="1"
+              gradientTransform="translate(52011.94 135563.12) rotate(-8.61) scale(220.48 -243.7)"
+              gradientUnits="userSpaceOnUse"
+            >
+              <stop offset=".22" stop-color="#a5f0e7" />
+              <stop offset="1" stop-color="#f7c7c7" stop-opacity="0" />
+            </radialGradient>
+            <clipPath id="nft-widget-def-clip">
+              <path
+                d="M175,216h348c4.42,0,8,3.58,8,8v144H167V224c0-4.42,3.58-8,8-8Z"
+              />
+            </clipPath>
+          </defs>
+          <g>
+            <rect
+              x="0"
+              y="0"
+              width="414"
+              height="564"
+              rx="24"
+              ry="24"
+              fill="#fff"
+            />
+            <g
+              class="nft-widget-graph__dummy-widget-small"
+              filter="url(#nft-widget-graph-blur)"
+            >
+              <rect
+                x="32"
+                y="380"
+                width="169.36"
+                height="90"
+                rx="12"
+                ry="12"
+                fill="url(#nft-widget-def-b)"
+              />
+              <rect
+                x="32"
+                y="380"
+                width="169.36"
+                height="90"
+                rx="12"
+                ry="12"
+                fill="url(#nft-widget-def-c)"
+              />
+              <rect
+                x="32"
+                y="380"
+                width="169.36"
+                height="90"
+                rx="12"
+                ry="12"
+                fill="url(#nft-widget-def-d)"
+                fill-opacity=".3"
+              />
+            </g>
+            <rect
+              v-for="(line, index) in lineProps"
+              :key="`nft-widget-graph__line-${index + 1}`"
+              class="nft-widget-graph__line"
+              v-bind="line"
+              x="32"
+              height="8"
+              rx="4"
+              ry="4"
+              fill="#ebebeb"
+            />
+          </g>
+        </svg>
+        <div
+          class="
+            w-full
+            mx-auto
+            mb-[24px]
+            laptop:pl-[16px] laptop:ml-[24px] laptop:absolute
+            bottom-0
+            right-0
+            laptop:scale-100
+            max-w-[435px]
+            nft-widget-graph__dummy-widget
+          "
+        >
+          <NFTWidget
+            class="shadow-lg"
+            v-bind="article"
+            :collect-button-label="widgetCollectButtonLabel"
+          />
+        </div>
+      </div>
+      <div
+        class="
+          nft-widget-graph__description
+          mt-[16px]
+          laptop:mt-[0px] laptop:max-w-[320px] laptop:mb-[70px] laptop:ml-[32px]
+          leading-[22px]
+        "
+        v-html="$t('about_nft_page_nft_widget_description')"
```

Comment:
Prefer i18n path than `v-html` if possible

Reply:
There are some bold texts in between the translated content, it seems `v-html` is the only [option](https://kazupon.github.io/vue-i18n/guide/formatting.html#html-formatting) without using the complicated interpolation